Library/Homebrew/os/mac/xcode.rb

```diff
@@ -105,7 +105,7 @@ def uncached_version
         # Xcode.version would always be non-nil. This is deprecated, and will
         # be removed in a future version. To remain compatible, guard usage of
         # Xcode.version with an Xcode.installed? check.
-        case DevelopmentTools.llvm_build_version.to_i
+        case DevelopmentTools.clang_version.to_i
```

Comment:
This cannot be simply replaced as `clang_version` is distinct from `llvm_build_version`. Unless this can be substituted with something else, we'll probably have to drop the entire `case` **except for** the contents of the `else` clause. As this is about inferring a fake Xcode version from the compilers provided on a CLT-only system, that's not a big loss.